apps/molgenis-viz/vite.config.js

```diff
@@ -2,6 +2,10 @@ import { defineConfig } from "vite";
 import path from "path";
 import vue from "@vitejs/plugin-vue";
 
+const host = "https://david-test.molgeniscloud.org";

```

Comment:
Is this a good default for everyone?

Reply:
I've run into issues where the schema or example data was deleted (probably due to molgenis upgrades), so I changed the host to one that doesn't change that much. At some point, it would be nice to have a fixed host for all vue applications.